### ISSUE TITLE
fix(android): new NativeEventEmitter() warning on RN 65+

### ISSUE THREAD

**Issue by Titozzz** [launchdarkly/react-native-client-sdk#116]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `launchdarkly-react-native-client-sdk@6.0.0` for the project I'm working on.

I initially though this was the reason this module wasn't "compatible" with RN 0.65+ but you released 6.0.0 and it's not fixed.
When used on android with react native >= 0.65, there is a warning.
This patch removes the warning. See theses PR to other libraries:
- https://github.com/thebergamo/react-native-fbsdk-next/pull/89/files
- https://github.com/invertase/react-native-firebase/pull/5616/files
- etc..

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/launchdarkly-react-native-client-sdk/android/src/main/java/com/launchdarkly/reactnative/LaunchdarklyReactNativeClientModule.java b/node_modules/launchdarkly-react-native-client-sdk/android/src/main/java/com/launchdarkly/reactnative/LaunchdarklyReactNativeClientModule.java
index a3cdb1b..23a94d4 100644
--- a/node_modules/launchdarkly-react-native-client-sdk/android/src/main/java/com/launchdarkly/reactnative/LaunchdarklyReactNativeClientModule.java
+++ b/node_modules/launchdarkly-react-native-client-sdk/android/src/main/java/com/launchdarkly/reactnative/LaunchdarklyReactNativeClientModule.java
@@ -187,6 +187,16 @@ public class LaunchdarklyReactNativeClientModule extends ReactContextBaseJavaMod
         return constants;
     }
 
+    @ReactMethod
+    public void addListener(String eventName) {
+        // Keep: Required for RN built in Event Emitter Calls.
+    }
+
+    @ReactMethod
+    public void removeListeners(Integer count) {
+        // Keep: Required for RN built in Event Emitter Calls.
+    }
+
     @ReactMethod
     public void configure(ReadableMap config, ReadableMap user, final Promise promise) {
         internalConfigure(config, user, null, promise);
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by louis-launchdarkly** [comment]
Hello @Titozzz, thank you for providing the instruction about how to patch out this warning. We will look into the detail of this warning. If we are going to apply this patch, because we will need to update the Android SDK and add that version to patch the RN SDK, it may take us some time to release. We will give you an update when we have the change ready.

_Filed Internal as 142071._

**Comment by louis-launchdarkly** [comment]
@Titozzz, thank you for pointing that out, I was too hasty to think because RN SDK wraps the Android SDK, all Java code changes need to go into the Android SDK (which is not true for the bridge file that you proposed the change).

Given you already have the fix, do you mind putting a PR? If you don't have time, I can start doing that on the private mirror.

**Comment by gwhelanLD** [comment]
@Titozzz,

Just wanted to update you that we've got the change merged in our internal private repo, so no need to PR this change. We'll have a release with this fix soon.

Thanks,
@gwhelanLD

**Comment by Titozzz** [comment]
Oh ok i just added it to my to-do yesterday, cool to know

**Comment by louis-launchdarkly** [comment]
Fixed in 6.1.0.
